EventFilter/TotemRawToDigi/test/analyze_vfat_frames.py

```diff
@@ -0,0 +1,43 @@
+import FWCore.ParameterSet.Config as cms
+import FWCore.ParameterSet.VarParsing as VarParsing
+
+process = cms.Process("AnalyzeVFATFrames")
+
+# default options
+options = VarParsing.VarParsing ('analysis')
+options.inputFiles= 'file:/afs/cern.ch/user/j/jkaspar/public/run268608_ls0001_streamA_StorageManager.root',
```

Comment:
https://twiki.cern.ch/twiki/bin/view/CMSPublic/WorkBookUsingT2Resources
https://twiki.cern.ch/twiki/bin/view/CMSPublic/WorkBookFileTransfer
https://twiki.cern.ch/twiki/bin/view/CMSPublic/WorkBookDataManagementBackground
https://twiki.cern.ch/twiki/bin/view/CMS/T2CHCERNEosTeams
might be useful